src/cli/cli.cpp

```diff
@@ -8322,6 +8409,11 @@ otError Interpreter::ProcessCommand(Arg aArgs[])
 #if OPENTHREAD_CONFIG_UPTIME_ENABLE
         CmdEntry("uptime"),
 #endif
+#if OPENTHREAD_FTD || OPENTHREAD_CONFIG_TMF_NETWORK_DIAG_MTD_ENABLE
+        CmdEntry("vendormodel"),
+        CmdEntry("vendorname"),
+        CmdEntry("vendorswversion"),
```

Comment:
just curious: Why not make `model`, `name` and `swversion` the subcommands of `vendor` command?

Reply:
I was expecting maybe slightly smaller code (since we would use lookup table and avoid `if()` checks) but seems like it does not impact much.

I changed it to sub-command model in new push.